docs/azure.rst

```diff
@@ -353,6 +353,11 @@ The following configuration options are available:
 ============================================== =================
 Name                                           Description
 ============================================== =================
+azure.azcopy.blockSize                          The block size (in MiB) when uploading to Azure Storage, and downloading from Azure Storage (default: , requires ``nf-azure@0.13.1``).
+azure.azcopy.blobTier                           The blob tier to be used while uploading to Azure Storage (default: , requires ``nf-azure@0.13.1``).
+azure.azcopy.putMD5                             Create an MD5 hash of each file during upload, and save the md5sum hash as the Content-MD5 property of the destination blob or file. (default: false)
+azure.azcopy.checkMD5                           Specifies how strictly MD5 hashes should be validated when downloading data. (default 'FailIfDifferent')
```

Comment:
Any reason to not make this the default?

Reply:
Paolo, this property is just surfacing the `azcopy` default of "FailIfDifferent" for information purpose. The users can opt to change if necessary.